paper/tsd2017.tex

```diff
@@ -40,11 +40,11 @@
 
 % TSD 2017: Put your title here (please, use capitalization, see e.g.
 % http://en.wikibooks.org/wiki/Basic_Book_Design/Capitalizing_Words_in_Titles)
-\title{Language-Independent Acoustic Modeling for Medieval Latin}
+\title{Language Specific Knowledge and Data Free Acoustic Modeling for Medieval Latin LVCSR}
```

Comment:
Én azért hagynám benne a 'language independent'-et, mert egy kifejezés, amit mát használtak arra (és még szofisztikáltabb módszerekre), amit mi csinálunk a cikkben. Nem ezt használni, és új nevet adni neki olyan, mintha nem olvasnánk szakirdalmat.

Ahogy én láttam,  'language-independent'-be beletartoznak a szakértői rendszerek, az independent inkább mintha azt jelentené, hogy a célnyelvtől független adatot használunk.